pyscf/adc/radc.py

```diff
@@ -573,9 +719,13 @@ def outcore_transform():
 
         eris = self.transform_integrals()
         
-        self.e_corr, self.t1, self.t2 = compute_amplitudes_energy(self, eris=eris, verbose=self.verbose)
-        self.e_tot = self.scf_energy + self.e_corr
+        if self.get_mpn_energy == True:
+            self.e_corr, self.t1, self.t2 = compute_amplitudes_energy(self, eris=eris, verbose=self.verbose)
+        else:
+            self.t1, self.t2 = compute_amplitudes(self, eris=eris)
+            self.e_corr = 0.0
```

Comment:
Probably should set e_corr to None if it's not computed

Reply:
Done